logging-modules/log4j2/src/main/java/com/baeldung/logging/log4j2threadinfo/Log4j2ThreadInfo.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.logging.log4j2threadinfo;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.stream.IntStream;
```

Comment:
Applying the formatter swap the order of the import statements.

Reply:
I applied it, but nothing changed in the import order. I think it is already formatted.